### PR TITLE
Added course show and manage

### PR DESCRIPTION
**Pull request by nhieu92**
Added course show and manage

### CODE REVIEW

**Comment by tranthuc141** [comment]
OK!

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+  email = "example-#{n+2}@hieunt.com"
+  password = "12345678"
+  User.create!(email: email,
+    password: password,
+    password_confirmation: password,
+    role: "normal")
+end
+
+30.times do |n|
+  name = "course-#{n+1}"
```
Comment (dieunb) [inline db/seeds.rb]:
```
name = "course-#{n + 1}"
```

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+  description = "Content -#{n+1}"
```
Comment (dieunb) [inline db/seeds.rb]:
```
"Content -#{n + 1}"
```

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create!(email: 'nhieu92@gmail.com',
+  password: "12345678",
+  password_confirmation: "12345678",
+  role: "admin")
+
+30.times do |n|
+  email = "example-#{n+2}@hieunt.com"
```
Comment (dieunb) [inline db/seeds.rb]: `{n + 2}`

**Pull request merged**